core/src/main/clojure/xtdb/log.clj

```diff
@@ -229,15 +231,17 @@
 
       (.getAsArrowStream rel))))
 
-(defmethod xtn/apply-config! ::memory-log [^Xtdb$Config config _ {:keys [instant-src]}]
+(defmethod xtn/apply-config! ::memory-log [^Xtdb$Config config _ {:keys [instant-src current-epoch]}]
```

Comment:
nit: `current` here is superfluous given I think we only ever deal with the current one? -> `epoch`?